### DISCUSSION TITLE
Table 3

### DISCUSSION THREAD

**Discussion by ltealdawson** [IBAHCM/RPiR#530] [Registration]
@rebeccamayer 
@LETUS-MUYAGA
@RamahKimbu

**Comment by LETUSMUYAGA** [comment]
My GitHub repository is at https://github.com/SBOHVM/LETUS-MUYAGASeries01.git

**Comment by ltealdawson** [comment]
My git repository is at: 
https://github.com/SBOHVM/ltealdawsonseries01.git

**Comment by RebeccaMayer** [comment]
my repository: https://github.com/SBOHVM/RebeccaMayerSeries01.git

**Comment by RamahKimbu** [comment]
my github repository is https://github.com/SBOHVM/RamadhaniSeries01.git

**Comment by mhuzzell** [comment]
Commenting so that table 3 has my username, for Series 2.

**Comment by RebeccaMayer** [comment]
my series 2 repository: https://github.com/SBOHVM/RebeccaMayerSeries02.git

**Comment by mhuzzell** [comment]
Here's my repository for Series 2: https://github.com/SBOHVM/mhuzzellSeries02.git

**Comment by LETUSMUYAGA** [comment]
My repository for Series 2 practical: https://github.com/SBOHVM/LETUSMUYAGASeries02.git

**Comment by mhuzzell** [comment]
Have any of you pushed the code for practical 2-1 yet? Asking because I don't see it in your repositories, but I had had trouble with mine, and I'm not sure if anyone else is as well? 

I had tried to push my code for 2-1 on Monday, and then it seemed to get stuck in some sort of nether-world where it wasn't available to upload but still wasn't appearing on Github. I was also getting an error message when I tried to look at it that said I was trying to push an unusually large file (I think because I'd included an html report), and asking me to confirm I wanted to do that, but with no option to actually confirm... Then when I came back here yesterday intending to ask about it, it had all appeared on my repository overnight.

**Reply by LETUSMUYAGA** [reply]
Hi @mhuzzell  I have already pushed my codes you can have a look at it 
Am not sure if we need to push the Html, I'll recheck the instructions and confirm this.

I have some issues which I'll submit in a few minutes

**Reply by mhuzzell** [reply]
I don't think we needed to push it. I just ticked all the boxes and then realised after hitting 'commit' that it included the html report.

**Reply by RebeccaMayer** [reply]
I'm having issues pushing mine as well, nothing is showing up in my repository. Might have to wait until class tomorrow.

**Reply by richardreeve** [reply]
That's okay to wait till class. It takes a bit of time to get used to doing this. It does get easier though, honest!

**Comment by RamahKimbu** [comment]
My series 2 repository; https://github.com/SBOHVM/RamahKimbuSeries02.git

**Comment by ltealdawson** [comment]
My series 2 repository: https://github.com/SBOHVM/ltealdawsonseries02.git
